### ISSUE TITLE
Replay and select "My Control" , then aircraft explode?

### ISSUE THREAD

**Issue by super-xxx** [NikolaiVChr/f16#566]
Hi ,
   Recently, when I used Replay System  I found a bug：
If there is a terrain obstruction between the last location and the new replay location, f16 will explode.
I don't know this bug is in nasal or in FlightGear .

**Comment by NikolaiVChr** [comment]
The replay system in F-16 is not really being maintained. Its very simple and barely working.
It is therefore expected that weird stuff like that happens.
We could put some effort into improving it, but that will also means it has to record more properties, which is heavy on the computer, and we would like to avoid that.

**Comment by Dan263737** [comment]
Yes, same problem here. I my scenario, it sometimes explode, sometimes it turns off

**Comment by Dan263737** [comment]
Update: after selecting My Controls, jet stops mid air and I cant get it to move or start,repair,anything
